hr_expense_petty_cash/data/account_account.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="petty_cash" model="account.account">
```

Comment:
We don't need to provide the account code. Accountant should be the person to add this manually into his/her chart of account.
This file can be removed.
Note: in the test script, you can use coding to create this account before run test.